ui/components/app/transaction-list/transaction-list.component.js

```diff
@@ -334,13 +334,64 @@ export default function TransactionList({
     setSelectedTransaction(transaction);
   }, []);
 
+  const getTransactionDisplayAmount = (transaction, userAddress) => {
```

Comment:
Note: in the future we might need to work on a design that allows more than one balance to change in a single transaction.

Reply:
Agree, design is too limited right now